src/NServiceBus.Core/OpenTelemetry/Metrics/Meters.cs

```diff
@@ -19,4 +19,7 @@ class Meters
 
     internal static readonly Histogram<double> MessageHandlerTime =
         NServiceBusMeter.CreateHistogram<double>(Metrics.MessageHandlerTime, "s", "The time in seconds for the execution of the business code.");
+
+    internal static readonly Histogram<double> CriticalTime =
+        NServiceBusMeter.CreateHistogram<double>(Metrics.CriticalTime, "s", "The time in seconds between when the message was sent until processed by the endpoint.");
```

Comment:
```suggestion
        NServiceBusMeter.CreateHistogram<double>(Metrics.CriticalTime, "s", "The time in seconds from when the message was sent to when it was processed.");
```

Reply:
I'm fine either way. The documentation states the following:

> The time between when a message is sent and when it is fully processed.

Shell we align the text here with docs?

```suggestion
        NServiceBusMeter.CreateHistogram<double>(Metrics.CriticalTime, "s", "The time between when a message is sent and when it is fully processed.");
```